src/ukify/ukify.py

```diff
@@ -833,14 +921,59 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
             value = self.metavar or self.argparse_dest().upper()
         return (section_name, key, value)
 
-
-CONFIG_ITEMS = [
+CONFIG_ITEMS_COMMON = [
     ConfigItem(
         '--version',
         action = 'version',
         version = f'ukify {__version__}',
     ),
 
+    ConfigItem(
+        ('--config', '-c'),
+        metavar = 'PATH',
+        help = 'configuration file',
+    ),
+
+    ConfigItem(
+        '--summary',
+        help = 'print parsed config and exit',
+        action = 'store_true',
+    ),
+]
+
+CONFIG_ITEMS_DISPLAY = [
+    ConfigItem(
+        'file',
+        nargs = '?',
```

Comment:
While looking at the default subparser problem, I noticed that this contradicts `apply_display_config`. If you make the argument mandatory, `apply_display_config` becomes a noop and if no argument is given it won't throw a backtrace anymore.

Reply:
Sorry I didn't get this one

Reply:
`?` means zero or one argument, but if you pass no argument, a `ValueError` will be thrown in in `apply_display_config`. This can be avoided by making this `nargs = 1` instead, but then the check in `apply_display_config` can be removed because the code is no longer reachable.

Reply:
Makes a lot of sense. thanks!